lib/trento/domain/cluster/commands/deregister_cluster_host.ex

```diff
@@ -0,0 +1,13 @@
+defmodule Trento.Domain.Commands.DeregisterClusterHost do
+  @moduledoc """
+    Deregister a cluster host
```

Comment:
Personally, I always found this wording a bit confusing as it sounds as if a "_cluster host_" is a special type of host.

What about something like "Unregister a host from a cluster" or "Remove a host from a cluster"?

Reply:
Though I'm not a huge fan of the "deregister" word myself, I'm all about commitment :smile: ... Let's go for now with that word and change it later. I'll make the second part of the sentence more clear though, I think you are right.